### PR TITLE
Implement deep link to shop online category screen

### PR DESCRIPTION
**Pull request by msalcala11**
The following deep link will now work: `bitpay://shoponline?category=charities%20%26%20nonprofits`

### CODE REVIEW

**Comment by cmgustavo** [comment]
I can see that the link is valid: 
```
2022-11-23 11:29:50.302290-0300 BitPayApp[29300:559909] [javascript] '[Info]', '[deeplink] valid: bitpay://shoponline?category=charities%20%26%20nonprofits'```
But it doesn't redirect to the category. Also it doesn't open the tab Shop.

File: src/utils/hooks/useDeeplinks.ts [hunk 1]
```diff
@@ -96,6 +98,7 @@ export const useUrlEventHandler = () => {
 export const useShopDeepLinkHandler = () => {
   const navigation = useNavigation();
   const availableGiftCards = useAppSelector(selectAvailableGiftCards);
+  const categories = useAppSelector(selectCategoriesWithIntegrations);
   const integrations = useAppSelector(selectIntegrations);
 
   const shopDeepLinkHandler = (
```
Comment (cmgustavo) [inline src/utils/hooks/useDeeplinks.ts:104]: For some reason, this function is never called.

Comment (msalcala11) [reply src/utils/hooks/useDeeplinks.ts]: Thanks for catching this. Should work now.

**Comment by msalcala11** [comment]
Thanks for the review and the video! Should work now.

**Review by cmgustavo — APPROVED** [review]
LGTM! Nice work 👍🏼

**Pull request merged**